pyproject.toml

```diff
@@ -19,6 +19,8 @@ from = "src"
 python = "^3.11"
 fastapi = "^0.104.1"
 uvicorn = "^0.24.0"
+tortoise-orm = "^0.20.0"
+aiodocker = "^0.21.0"
```

Comment:
Seems like aiodocker doesn't pass 3.12

but you can try locking aiohttp to https://pypi.org/project/aiohttp/3.9.0rc0/
to pass 3.12, either way the stable wheels for aiohttp on 3.12 should be released soon so we can ignore 3.12 for now

Reply:
Will wait for the stable wheels for aiohttp then. Leaving as is for now.